utils/elftocpm65/elftocpm65.cc

```diff
@@ -0,0 +1,285 @@
+#include <fcntl.h>
+#include <fmt/format.h>
```

Comment:
Probably shouldn't have a fmt dep either; at the very least, this will break our builders.

Reply:
Sob. Removed. I've also changed it to work in post-link-tool mode, which makes argument parsing easier, so I got rid of getopt too.